wandb/apis/public/artifacts.py

```diff
@@ -439,34 +463,12 @@ def change_type(self, new_type: str) -> None:
         }
         mutation = gql(template)
         self.client.execute(mutation, variable_values=variable_values)
-        self.type = new_type
+        self._saved_type = new_type
+        self._type = new_type
 
-    @normalize_exceptions
-    def is_sequence(self) -> bool:
-        """Return True if this is a sequence."""
-        query = gql(
-            """
-            query FindSequence($entity: String!, $project: String!, $collection: String!, $type: String!) {
-                project(name: $project, entityName: $entity) {
-                    artifactType(name: $type) {
-                        __typename
-                        artifactSequence(name: $collection) {
-                            __typename
-                        }
-                    }
-                }
-            }
-            """
-        )
-        variables = {
-            "entity": self.entity,
-            "project": self.project,
-            "collection": self.name,
-            "type": self.type,
-        }
-        res = self.client.execute(query, variable_values=variables)
-        sequence = res["project"]["artifactType"]["artifactSequence"]
-        return sequence is not None and sequence["__typename"] == "ArtifactSequence"
+    def is_sequence(self):
```

Comment:
typo: return type hint may have been inadvertently dropped, consider restoring:
```suggestion
    def is_sequence(self) -> bool:
```

Reply:
thanks for the catch!